build.sbt

```diff
@@ -163,6 +163,8 @@ lazy val interopTests = Project(id = "interop-tests", base = file("interop-tests
     ReflectiveCodeGen.generatedLanguages := Seq("Scala", "Java"),
     ReflectiveCodeGen.extraGenerators := Seq("ScalaMarshallersCodeGenerator"),
     ReflectiveCodeGen.codeGeneratorSettings ++= Seq("server_power_apis"),
+    // grpc 1.53.0 brings in extra unnecessary proto files that cause build issues
```

Comment:
What kinds of issues? Should we create a GitHub issue for these and link it here?

Reply:
just for reference, the build failures look like https://github.com/apache/incubator-pekko-grpc/actions/runs/6065657613/job/16455401558

The opencensus protos that cause the issues are imported by envoy protos. We don't seem to need these envoy protos. 

Since the issue only popped up after upgrading the io.grpc libs, I have to presume that the protos are being brought in by those jars.

Reply:
the extra envoy and opencensus protos appear to have been added to grpc-interop-testing jar between the 1.48.1 and 1.53.0 releases